$NetBSD: patch-meson.build,v 1.3 2019/07/29 19:40:51 jperkin Exp $

-D_XOPEN_SOURCE=1 implies POSIX.1 but we need at least 500 (SUSv2) for
struct timeval. Fixes build on SunOS.
https://gitlab.gnome.org/GNOME/gdk-pixbuf/issues/128

--- meson.build.orig	2019-02-28 16:22:57.000000000 +0000
+++ meson.build
@@ -8,7 +8,7 @@ project('gdk-pixbuf', 'c',
         ],
         meson_version: '>= 0.46.0')
 
-add_project_arguments([ '-D_POSIX_C_SOURCE=200809L', '-D_DEFAULT_SOURCE', '-D_XOPEN_SOURCE' ], language: 'c')
+add_project_arguments([ '-D_POSIX_C_SOURCE=200809L', '-D_DEFAULT_SOURCE', '-D_XOPEN_SOURCE=600' ], language: 'c')
 
 cc = meson.get_compiler('c')
 host_system = host_machine.system()
